9.2-jre7/Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN set -xe \
 	&& curl -SL "$JETTY_TGZ_URL.asc" -o jetty.tar.gz.asc \
 	&& export GNUPGHOME="$(mktemp -d)" \
 	&& for key in $JETTY_GPG_KEYS; do \
-		gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key"; done \
+		gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys "$key"; done \
 	&& gpg --batch --verify jetty.tar.gz.asc jetty.tar.gz \
 	&& rm -rf "$GNUPGHOME" \
```

Comment:
Any reason not to add `gpgconf --kill all` to this `Dockerfile`? I suppose it is for JRE 7, so this Dockerfile should probably just be retired.

Reply:
That'd be because this is based on Jessie which still uses GnuPG 1 so doesn't have `dirmngr` or `gpgconf` (nor daemons that get started automatically and need killing).